### PR TITLE
Tenant request schemas

### PR DESCRIPTION
**Pull request by niklas-la**

### CODE REVIEW

File: src/assets/server/rest/v1/schemas/tenant/tenant-update-req-super-admin.json [hunk 1]
```diff
@@ -2,143 +2,13 @@
...
-        }
+        "id": { "$ref": "common.json#/definitions/id" },
+        "name": { "$ref": "common.json#/definitions/name"},
+        "email": { "$ref": "common.json#/definitions/email" },
+        "subdomain": { "$ref": "common.json#/definitions/subdomain" },
+        "logo": { "$ref": "common.json#/definitions/logo" },
+        "address": { "$ref": "common.json#/definitions/address" },
+        "components": { "$ref": "components.json#/definitions/components" }
     },
-    "required": ["id", "name", "email", "subdomain"]
```
Comment (OliveGerste) [inline src/assets/server/rest/v1/schemas/tenant/tenant-update-req-super-admin.json:143]: The id is mandatory on the Update request. I think you have to take it back in.

File: src/assets/server/rest/v1/schemas/tenant/tenant-get-logo-req-super-admin.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "title": "Tenant Logo",
```
Comment (OliveGerste) [inline src/assets/server/rest/v1/schemas/tenant/tenant-get-logo-req-super-admin.json:2]: I would name it consistent. So I would suggest 'TenantLogo Get' in this case.

File: src/server/rest/v1/validator/TenantValidation.ts [hunk 3]
```diff
@@ -40,6 +46,24 @@ export default class TenantValidator extends SchemaValidator {
...
+  public validateGetLogoReqSuperAdmin(request: any): string {
+    // Validate schema
+    this.validate(this._tenantGetLogoReqSuperAdmin, request);
+    return request.ID;
+  }
+
+  public validateTenantGetReqSuperAdmin(request: any): string {
+    // Validate schema
+    this.validate(this._tenantGetReqSuperAdmin, request);
+    return request;
```
Comment (OliveGerste) [inline src/server/rest/v1/validator/TenantValidation.ts:64]: You return the whole request here. Not the id, which you actually need where you call it. As a result you are not able to edit a tenant anymore.

File: src/assets/server/rest/v1/schemas/tenant/tenant-get-req-super-admin.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
+{
+    "title": "Tenant Request",
```
Comment (OliveGerste) [inline src/assets/server/rest/v1/schemas/tenant/tenant-get-req-super-admin.json:2]: Same as above. I would name it 'Tenant Get'

**Review by OliveGerste — CHANGES_REQUESTED** [review]

File: src/assets/server/rest/v1/schemas/common/common.json [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+                    "sanitize": "mongo"
+                },
+                "coordinates": {
+                    "type": "array",
+                    "sanitize": "mongo"
+                }
+            }
+        }
+    }
+}
```
Comment (OliveGerste) [inline src/assets/server/rest/v1/schemas/common/common.json:69]: Add new lines at the end of the files.

**Comment by LucasBrazi06** [comment]
Seems that issues were fixed but still in the **Dev cross-review** pipeline, normal?

**Comment by OliveGerste** [comment]
> @OliveGerste
> Seems that issues were fixed but still in the **Dev cross-review** pipeline, normal?

@LucasBrazi06 No sorry, I moved it.

**Pull request merged**